ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/CheckDetailsPage.tsx

```diff
@@ -1,29 +1,31 @@
 import React from 'react';
-import { useHistory, useParams } from 'react-router-dom';
-import { Button, PageSection } from '@patternfly/react-core';
+import { Breadcrumb, BreadcrumbItem, Divider, PageSection } from '@patternfly/react-core';
+import { useParams } from 'react-router-dom';
 
+import PageTitle from 'Components/PageTitle';
+import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
 import { complianceEnhancedCoveragePath } from 'routePaths';
 
-function CheckDetailsPage() {
-    const history = useHistory();
-    const { profileName } = useParams();
+function CheckDetails() {
+    const { checkName, profileName } = useParams();
+
+    const complianceCoverageChecksURL = `${complianceEnhancedCoveragePath}/profiles/${profileName}/checks`;
+
     return (
         <>
-            <PageSection variant="light">
-                <div>CheckDetailsPage</div>
-                <Button
-                    onClick={() => {
-                        history.push(
-                            `${complianceEnhancedCoveragePath}/profiles/${profileName}/checks`
-                        );
-                    }}
-                    variant="primary"
-                >
-                    Go to all checks (ProfileChecksPage)
-                </Button>
+            <PageTitle title="Compliance coverage - Check" />
```

Comment:
Do we want to centralize titles in one location for compliance? Might be easier to change them if we ever needed to in the future.

Reply:
I go back and forth on this one. Seems a bit like unnecessary abstraction but I understand the argument as to why it should be centralized.